README.md

```diff
@@ -116,6 +116,16 @@ If you want review to appear on pull request diff, instead of comments:
 $ PRONTO_GITHUB_ACCESS_TOKEN=token pronto run -f github_pr_review -c origin/master
 ```
 
+All the comments will now not be published into a single PR review, but separated into a number of PRs.
```

Comment:
@dimitrovv my only feedback is the wording here which is a little confusing to me. We don't mean to say it will create new PRs but will only leave _N_ number of feedback per Pronto run, right?

If I create a PR with 100 errors, Pronto will report the first 30.
If I fix them, and push a new commit to my PR, Pronto will report the next 30.

Is that understanding correct? If so, I think we could rephrase this as such:

> In order to avoid rate limiting by providers, Pronto publishes a limited number of comments per execution. By default, this is limited to the first 30 reported issues. You can change the number of reported issues either through the `.pronto.yml` file or the `PRONTO_WARNINGS_PER_REVIEW` environment variable.

Reply:
> @dimitrovv my only feedback is the wording here which is a little confusing to me. 

Yes, I didn't define it the right way, will fix

---

> If I create a PR with 100 errors, Pronto will report the first 30.
If I fix them, and push a new commit to my PR, Pronto will report the next 30.

Actually, it will post all N comments, but separated into X number of PR reviews (X = N / PRONTO_WARNINGS_PER_REVIEW). So all the comments should be published to Github only with a single pronto run

Reply:
Thank you @dimitrovv

Reply:
@doomspork, I've just tried to formulate the explanations around the new setting a bit better. Could you take a look when possible?